dev/devicelab/lib/tasks/plugin_tests.dart

```diff
@@ -103,4 +105,53 @@ class FlutterProject {
     }
     rmTree(parent);
   }
+
+}
+
+class _FlutterPlugin extends _FlutterProject {
+  _FlutterPlugin(Directory parent, String name) : super(parent, name);
```

Comment:
Is it possible to have everything in the `_FlutterProject` class?
It appears you can define a create method with `template`, `project_name/binary_name` arguments, and simply create:
- plugin by calling `create('plugin', 'plugintest', ...)`;
- app by calling `create('app', 'plugintestapp')`.

Reply:
I could go either way on this. Since it's not a public API and is confined to this file it's easy to change later. I've consolidated everything into one class since you seem to prefer that.